### DISCUSSION TITLE
Movimintos de los ejes y trabajo del laser

### DISCUSSION THREAD

**Discussion by riopar10** [arkypita/LaserGRBL#1494] [Q&A]
Estos son los movimientos que tiene mi CNC 3018 PRO - LASER CHINO 5.5 W, SUPUESTAMENTE.
$3 - 8 , ( X BIEN - Y BIEN - Z BIEN ,LASER FORMA ESPEJO)

$3 - 7 , ( MAL - Y MAL - Z MAL ,FORMA ESPEJO)

$3 - 6 , ( BIEN - Y MAL - Z MAL ,LASER BIEN)

$3 - 5 , ( X MAL - Y BIEN - Z MAL ,LASER FORMA ESPEJO)

$3 - 4 , ( X BIEN - Y BIEN - Z MAL ,LASER FORMA ESPEJO)

$3 - 3 , ( X MAL - Y MAL - Z BIEN ,LASER FORMA ESPEJO)

$3 - 2 , ( X BIEN - Y MAL - Z BIEN ,LASER BIEN)

$3 - 1 , ( X MAL - Y BIEN - Z BIEN ,LASER FORMA ESPEJO)

Es posible solucionar el modo 8 que no trabaje en forma de espejo.
Gracias

**Comment by BeatSlayer** [comment]
You need to invert your laser ttl.

Google it, it's relatively easy to solve.
